src/main/java/org/cryptomator/common/mount/MountModule.java

```diff
@@ -28,4 +31,12 @@ static ObservableValue<MountService> provideDefaultMountService(List<MountServic
 				fallbackProvider);
 	}
 
+
```

Comment:
here seems to be an extra empty line

Reply:
removed in 706d2b51be13e9ee715c012ed374ab4b113b0265